Readme.md

```diff
@@ -47,6 +47,7 @@ sent to the client (optional)
 * `contentType`: If the content to be sent is anything other than
 `application/json`, then the `contentType` property must be set (optional)
 * `bearerType`: string specifying the Bearer string (optional)
+* `bearerTypeCaseSensitive`: Configure if the bearerType should be case sensitive. Default is set to true. (optional)
```

Comment:
How about using `specCompliance` which accepts `rfc6749` and `rfc6750`.
This terms more accurate since the case sensitivity is all about following which spec.

Reply:
Can we have a simpler name?

1. I didnt find in either RFCs, that the bearerType can be case insenstive. 
2. I would be potentially forced to describe into the Readme in what each rfc differs.

Cant we do it like typescript and strict option? `strict` and `strictBearerTypeCaseSensitive` ?

Reply:
> I didnt find in either RFCs, that the bearerType can be case insenstive.

Yes, but the option can be extendible to not only about case sensitivity. It is about how we follow the spec.

> I would be potentially forced to describe into the Readme in what each rfc differs.

That is my intention, we need to describe why this option exist.
I think linking to both RFC (with a name) is enough. For example,

```readme
`specCompliance`: Configure how this plugin follow the spec. Accept either [`rfc6749`](https://datatracker.ietf.org/doc/html/rfc6749) or [`rfc`6750`](https://datatracker.ietf.org/doc/html/rfc6750). Default is set to `rfc6750`.

`rfc6749` is about the generic OAuth2.0 protocol which allows token type to be case-insensitive.
`rfc6750` is about the Bearer Token Usage which forces the token type to be exact match.
```

Reply:
done